src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/tests/test_webapp_commands.py

```diff
@@ -765,6 +765,27 @@ def test_functionapp_consumption_e2e(self, resource_group, storage_account):
         self.cmd('functionapp delete -g {} -n {}'.format(resource_group, functionapp_name))
 
 
+class FunctionAppOnLinux(ScenarioTest):
+    @ResourceGroupPreparer(location='japanwest')
```

Comment:
Why `japanwest`? The recommended location is `southcentralus`, which has lower traffic but is much closer to us.

Reply:
thanks, done!